sdk/keyvault/src/keyvault_client.rs

```diff
@@ -0,0 +1,39 @@
+use std::sync::Arc;
+
+use azure_core::auth::TokenCredential;
+use azure_svc_keyvault::{
+    create_key::Response,
+    models::{
+        key_create_parameters::{Crv, Kty},
+        KeyAttributes, KeyCreateParameters, KeyReleasePolicy,
+    },
+};
+
+pub struct KeyvaultClient {
```

Comment:
We already have Key Vault clients for keys, secrets, and certs. Why add another one that isn't scoped like all our other languages? We want our SDKs to be idiomatic, but still similarly designed.

Reply:
This looks to be for #803, where we look to migrate our SDKs to be on top of the generated service crates.

Reply:
Why is it under `sdk/keyvault` then? The structures of SDKs should remain idiomatic but consistent with all other languages' Azure SDKs.